includes/main.js

```diff
@@ -28,39 +28,53 @@ function initiateApp(){
 }
 function makeGallery(imageArray){
 	//use loops and jquery dom creation to make the html structure inside the #gallery section
-
 	//create a loop to go through the images in the imageArray
-		//create the elements needed for each picture, store the elements in variable
+		for (var imageArrayIndex = 0; imageArrayIndex < imageArray.length; imageArrayIndex++){
+			//$("#gallery").append(imageArray); //css class to append images
 
-		//attach a click handler to the figure you create.  call the "displayImage" function.  
-
-		//append the element to the #gallery section
-	
+		//create the elements needed for each picture, store the elements in variable
+			var fig = $('<figure>', {
+				class: "imageGallery col-xs-12 col-sm-6 col-md-4",
+				style: "background-image:url('"+ imageArray[imageArrayIndex]+"')"
```

Comment:
I wouldn't normally want to modify the style attribute like this, I would rather use the css method to edit the element.

```
var fig = $('<figure>', {
    class: "imageGallery col-xs-12 col-sm-6 col-md-4",
    css: {
         'background-image': 'url('+imageArray[imageIndex]+')'
    }
});
````